pkg/cloud/baremetal/actuators/machine/actuator.go

```diff
@@ -54,11 +54,13 @@ const (
 	ProviderName = "baremetal"
 	// HostAnnotation is the key for an annotation that should go on a Machine to
 	// reference what BareMetalHost it corresponds to.
-	HostAnnotation                = "metal3.io/BareMetalHost"
-	requeueAfter                  = time.Second * 30
-	externalRemediationAnnotation = "host.metal3.io/external-remediation"
-	poweredOffForRemediation      = "remediation.metal3.io/powered-off-for-remediation"
-	requestPowerOffAnnotation     = "reboot.metal3.io/capbm-requested-power-off"
+	HostAnnotation                  = "metal3.io/BareMetalHost"
+	requeueAfter                    = time.Second * 30
+	externalRemediationAnnotation   = "host.metal3.io/external-remediation"
+	poweredOffForRemediation        = "remediation.metal3.io/powered-off-for-remediation"
+	requestPowerOffAnnotation       = "reboot.metal3.io/capbm-requested-power-off"
+	nodeLabelsBackupAnnotation      = "remediation.metal3.io/nodeLabelsBackup"
```

Comment:
The convention is for label and annotation keys to be all lowercase.

Reply:
fixed. thanks